util/validators.ts

```diff
@@ -32,6 +32,22 @@ export const validateApiList = (val: string): string => {
   return val;
 };
 
+export const validateProductionApiList = (val: string): string => {
```

Comment:
per my comment about the API arrays, I would say we can use the same function here. don't need to DRY if it's not intuitive but in this case they are addressing the exact same problem, imo.